vars/buildDockerAndPublishImage.groovy

```diff
@@ -178,24 +182,29 @@ def call(String imageName, Map userConfig=[:]) {
             final String ghVersion = '2.5.2' // TODO: track with updatecli
             final String platformId = "${operatingSystem}_${cpuArch}"
             final String ghUrl = "https://github.com/cli/cli/releases/download/v${ghVersion}/gh_${ghVersion}_${platformId}.tar.gz"
-
-            withEnv(["platform_id=${platformId}", "gh_url=${ghUrl}"]) {
+            final String ghReleasesApiUri = "/repos/${org}/${repository}/releases"
+            withEnv([
+              "GH_URL=${ghUrl}",
+              "GH_RELEASES_API_URI=${ghReleasesApiUri}",
+            ]) {
               sh '''
               if ! command -v gh 2>/dev/null >/dev/null
               then
                 echo "INFO: No gh binary found: Installing it from ${ghUrl}."
                 temp_dir="$(mktemp -d)"
-                curl --silent --show-error --location --output "${temp_dir}/gh.tgz" "${gh_url}"
+                curl --silent --show-error --location --output "${temp_dir}/gh.tgz" "${GH_URL}"
```

Comment:
Not the point of the PR, but ❤️ long opts :) Very easily readable

Reply:
that is a motto. I'm not doing it enough, but long flags is so much help for people reading scripts \o/